qt/scientific_interfaces/DynamicPDF/DPDFFitControl.cpp

```diff
@@ -16,6 +16,7 @@
 #include "MantidKernel/Logger.h"
 #include "MantidQtWidgets/Common/AlgorithmRunner.h"
 #include "MantidQtWidgets/Common/FitOptionsBrowser.h"
+#include "MantidQtWidgets/Common/FittingMode.h"
```

Comment:
can you point me at some tests I can run for this interface?

Reply:
This only exists in Qt4 so it seems it is unused and can't be tested. If the similar `FittingMode` changes in indirect and muon work then this should also be ok.